specifications/api/swagger.yaml

```diff
@@ -2205,29 +2205,40 @@ components:
             The absolute minimum quantity of ada required for a new transaction
             output created with the wallet.
 
+            Warning: This field may be deprecated or removed in future versions
+            of the wallet.
```

Comment:
Btw, is "this field may be deprecated [..] in future versions" too indirect, like an announcement of an announcement? I suggested something similar as well, as I wasn't sure if we could deprecate it without having a some kind of replacement. But arguably it might not matter, and calling it deprecated now might be more truthful?

But regardless, 👍

Reply:
> Btw, is "this field may be deprecated [..] in future versions" too indirect, like an announcement of an announcement? I suggested something similar as well, as I wasn't sure if we could deprecate it without having a some kind of replacement. But arguably it might not matter, and calling it deprecated now might be more truthful?
> 
> But regardless, +1

I think we're actually planning to discuss this at tomorrow's dev meeting. Well I've put it on the agenda, at least. 😄